src/Stratis.Bitcoin/Builder/FullNodeBuilder.cs

```diff
@@ -161,6 +162,18 @@ public IFullNode Build()
 
             this.Services = this.BuildServices();
 
+            if (this.NodeSettings.PrintHelpAndExit)
```

Comment:
Shouldn't this be before this.BuildServices?

Reply:
Prior to `this.BuildServices` the `this.Features.FeatureRegistrations` collection is empty.